.travis.yml

```diff
@@ -27,19 +27,28 @@ matrix:
     - make lint
     - make -j4 || exit 1
     - valgrind --tool=memcheck --leak-check=full --error-exitcode=1 ctest
+    env: TRAVIS_LANG=C++
   - os: osx
+    language: cpp
     compiler: clang
-    addons:
+    before_script:
+    - source $FEATHER_CPP/ci/before_script_travis.sh
+    - cmake -DCMAKE_CXX_FLAGS="-Werror" $FEATHER_CPP
+    script:
+    - make -j4 || exit 1
+    - ctest
+    env: TRAVIS_LANG=C++
+  - os: linux
+    language: r
+    compiler: gcc
+    before_install:
+    - cd R
+    - ./configure
+    after_success:
+    - Rscript -e 'install.packages("covr"); covr::codecov()'
```

Comment:
I think this becomes a non-issue if you add coverage analysis to the other languages. We may turn off R coverage for now to avoid this distraction, or perhaps you turn off the PR comments in codecov.